### DISCUSSION TITLE
How change TOTAL?

### DISCUSSION THREAD

**Discussion by 7lano** [tronikos/esphome-magnetometer-water-gas-meter#29] [Q&A] [answered]
Hi!
I just started using this project.

I see a variable "new_total" in esphome-magnetometer.yaml

How can I use this variable? Where in my esphome yaml file do I add this value?

**Comment by tronikos** [comment, marked as answer]
In Home Assistant, developer tools, actions, run: 
```
action: esphome.water_meter_set_total
data:
  new_total: 100
```

Replace 100 with the desired value

**Reply by 7lano** [reply]
Thank you! ♥

**Reply by xeechou** [reply]
> Thank you! ♥

after you set the totals. Does it continue matches the readings from the water meter? Mine is slowly drifting away from it though.

**Comment by wiremasteredm** [comment]
Hi,

After 2 weeks of playing with the sensor, it finaly works but I need to reset everything to 0 . How can I do that?
It would be nice to set the initial total to what my meter is showing. Any ideeas?

**Reply by tronikos** [reply]
In Home Assistant, developer tools, actions, call: `esphome.water_meter_set_total`
